Features/Stories/Fieldtypes/CheckboxFieldType.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Test the validations done on fields from Editorial Interface - Checkbox fieldtype
+
+    @javascript
+    Scenario: Publish with checkbox checked
+        Given I am logged in as admin on PlatformUI
+        And a Content Type exists with identifier "Checkbox" in Group with identifier "Content" with fields:
```

Comment:
It is possible to use the default Content Type 'Content' if not specified, the sentence is just missing I will add it ASAP.